2010-08-19  Andreas Schwab  <schwab@redhat.com>

	* sysdeps/i386/i686/multiarch/strspn.S [!SHARED]: Fix SSE4.2 check.

Index: glibc-2.12-2-gc4ccff1/sysdeps/i386/i686/multiarch/strspn.S
===================================================================
--- glibc-2.12-2-gc4ccff1.orig/sysdeps/i386/i686/multiarch/strspn.S
+++ glibc-2.12-2-gc4ccff1/sysdeps/i386/i686/multiarch/strspn.S
@@ -65,7 +65,7 @@ ENTRY(strspn)
 	jne	1f
 	call	__init_cpu_features
 1:	leal	__strspn_ia32, %eax
-	testl	$index_SSE2, CPUID_OFFSET+index_SSE4_2+__cpu_features
+	testl	$bit_SSE4_2, CPUID_OFFSET+index_SSE4_2+__cpu_features
 	jz	2f
 	leal	__strspn_sse42, %eax
 2:	ret
